proof/refine/ARM/TcbAcc_R.thy

```diff
@@ -2367,8 +2367,20 @@ lemma get_sc_refill_ready_sp:
            \<and> P s\<rbrace>"
   by (wpsimp simp: obj_at_def)
 
-lemma rescheduleRequired_corres:
-  "corres dc (valid_tcbs and weak_valid_sched_action and pspace_aligned and pspace_distinct)
+definition weaker_valid_sched_action where
```

Comment:
If this is really needed, consider moving it somewhere like `Invariants_H.thy`

Reply:
It's needed, but only for this proof. Does anyone else have an opinion on this?

Reply:
If it's only needed for local proofs, then I think it's ok to leave the definition here. Thanks for clarifying :+1:

Reply:
Maybe a line of comment explaining what this is would be good to have :)